website/docs/releases.mdx

```diff
@@ -44,13 +44,13 @@ config
 
 #### Minor fixes
 
-- Clicking on objects resulting in crashed UI
+**Clicking on objects resulting in crashed UI**
 We highly recommend users upgrade to v0.12.0 from 0.11.0, as unfortunately a bug slipped in where clicking on objects in the details tab for reconciled resources often resulted in the UI displaying a raw error message.
```

Comment:
We can't have a "Minor fix" with the suggestion "we highly recommend users upgrade" due to it. 

I would either remove the recommendation for now or move to the Highlights with something like "We highly recommend users of v0.11.0 upgrade to this version as it includes fixes for a number of UI issues, depending on when 0.12.1 is expected to land as this will contain similar guidance for the provider issue. My preference would be to move to highlights as the provider issue is less significant.

Then the error description could be more simple as "Clicking on certain objects in the details tab..."

Additionally suggest we rename Minor Fixes to Bug Fixes.

Reply:
Multi-Flux is too internally facing.

Would rename as "Support for multiple Flux instances on a single cluster" with a simple description. Note you will also either need to add the referenced screenshot or remove the reference

Reply:
Done. Thanks.